pandas/tests/arithmetic/test_period.py

```diff
@@ -728,13 +728,13 @@ def test_pi_add_sub_td64_array_non_tick_raises(self):
         tdi = pd.TimedeltaIndex(["-1 Day", "-1 Day", "-1 Day"])
         tdarr = tdi.values
 
-        msg = r"Input has different freq=None from PeriodArray\(freq=Q-DEC\)"
-        with pytest.raises(IncompatibleFrequency, match=msg):
+        msg = r"Cannot add or subtract timedelta64\[ns\] dtype from period\[Q-DEC\]"
+        with pytest.raises(TypeError, match=msg):
```

Comment:
doesn't IncompatibleFrequency inherit from ValueError? so this is technically an api change (ok though)

Reply:
@jbrockmendel ?

Reply:
yes, this is a change.  in the whatsnew i called it a bugfix